### DISCUSSION TITLE
Invoking make error

### DISCUSSION THREAD

**Discussion by ghost** [facebookresearch/faiss#3691] [Q&A]
# Summary

I am working on Windows and face the issue on following the set up from https://github.com/facebookresearch/faiss/blob/main/INSTALL.md.
I checked step 1 (cmake -B build -DFAISS_ENABLE_GPU=OFF -DBLA_VENDOR=Intel10_64_dyn -DMKL_LIBRARIES=C:\Users\jykan\AppData\Local\Programs\Python\Python312\Lib\site-packages/mkl/libs -DFAISS_ENABLE_PYTHON=OFF -DBUILD_TESTING=ON .) worked well. 

However, in step 2, I ran (make -C build -j faiss) and I got this message
```
make: Entering directory '/c/Users/jykan/Documents/faiss/build'
make: *** No targets specified and no makefile found.  Stop.
make: Leaving directory '/c/Users/jykan/Documents/faiss/build'
``` 
when I am trying to invoke make in faiss folder.

# Platform

OS:  Windows11

Faiss version: 

Installed from:  source

Faiss compilation options:  using MKL
cmake -B build -DFAISS_ENABLE_GPU=OFF -DBLA_VENDOR=Intel10_64_dyn -DMKL_LIBRARIES=C:\Users\jykan\AppData\Local\Programs\Python\Python312\Lib\site-packages/mkl/libs -DFAISS_ENABLE_PYTHON=OFF -DBUILD_TESTING=ON .

Running on:
- [x] CPU
- [ ] GPU

Interface: 
- [x] C++
- [ ] Python

# Reproduction instructions

I am working on Windows and face the issue on following the set up from https://github.com/facebookresearch/faiss/blob/main/INSTALL.md.
I checked step 1 (cmake -B build -DFAISS_ENABLE_GPU=OFF -DBLA_VENDOR=Intel10_64_dyn -DMKL_LIBRARIES=C:\Users\jykan\AppData\Local\Programs\Python\Python312\Lib\site-packages/mkl/libs -DFAISS_ENABLE_PYTHON=OFF -DBUILD_TESTING=ON .) worked well. 

However, in step 2, I ran (make -C build -j faiss) and I got this message
```
make: Entering directory '/c/Users/jykan/Documents/faiss/build'
make: *** No targets specified and no makefile found.  Stop.
make: Leaving directory '/c/Users/jykan/Documents/faiss/build'
``` 
when I am trying to invoke make in faiss folder.

**Comment by alexanderguzhva** [comment]
@Jae0Kang most likely, you need to put a number after `j`, so please try something like `make -C build -j 8 faiss` instead of `make -C build -j faiss`. Basically, `-j` parameter in `make` implies the number of `CPU cores` used during the compilation process. Maybe, there is an incompatibility for Windows.

**Comment by junjieqi** [comment]
@Jae0Kang have you tried on this one? 

> @Jae0Kang most likely, you need to put a number after `j`, so please try something like `make -C build -j 8 faiss` instead of `make -C build -j faiss`. Basically, `-j` parameter in `make` implies the number of `CPU cores` used during the compilation process. Maybe, there is an incompatibility for Windows.

**Comment by ghost** [comment]
I fixed it. Thank you for your help!
